cas-server-core-services/src/main/java/org/jasig/cas/services/DefaultServicesManagerImpl.java

```diff
@@ -140,7 +141,7 @@ public RegisteredService findServiceBy(final long id) {
      *
      * @return the tree set
      */
-    public TreeSet<RegisteredService> convertToTreeSet() {
+    public Collection<RegisteredService> convertToTreeSet() {
```

Comment:
Another IDE analysis rule... doesn't like concrete classes used for collections. I'll change it back.